Formula/sc-im.rb

```diff
@@ -0,0 +1,27 @@
+class ScIm < Formula
+  desc "Spreadsheet program for the terminal, using ncurses"
+  homepage "https://github.com/andmarti1424/sc-im"
+  url "https://github.com/andmarti1424/sc-im/archive/v0.6.0.tar.gz"
+  sha256 "5da644d380ab3752de283b83cce18c3ba12b068d0762c44193c34367a0dcbc38"
+  head "https://github.com/andmarti1424/sc-im.git", :branch => "freeze"
+
+  depends_on "ncurses"
+
+  def install
+    cd "src" do
+      system "make", "prefix=#{prefix}"
+      system "make", "prefix=#{prefix}", "install"
+    end
+  end
+
+  test do
+    input = <<-EOS.undent
+      let A1=1+1
+      getnum A1
+    EOS
+    output = pipe_output(
+      "#{bin}/scim --nocurses --quit_afterload 2>/dev/null", input
```

Comment:
I think the need for 2>/dev/null may be a bug in brew and/or CI. This just showed up on https://github.com/Homebrew/homebrew-core/pull/19321

Reply:
It’s only a CI issue as far as the CI runs in a non-interactive shell, and sc-im doesn’t seem to support that. I’ve been able to reproduce the issue in any non-interactive shell, e. g. when I run the following script in my text editor:

```
/usr/local/Cellar/sc-im/0.6.0/bin/scim \
  --nocurses --quit_afterload \
    <<< $'let A1=1+1\ngetnum A1'
```

![image](https://user-images.githubusercontent.com/1239874/31440853-a3a73150-ae91-11e7-85e1-a349d2802fd0.png)

Reply:
Yeah, it's looking like the issue with https://github.com/Homebrew/homebrew-core/pull/19321 is a regression.